pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -85,6 +85,32 @@ spec:
           value: $(params.snapshot)
         - name: requireInternalServices
           value: "true"
+    - name: reduce-snapshot
```

Comment:
Last file to fix. This depends on collect-data, so you can put it after collect-data in the file? collect-data is defined on line 114, just after this. So ideally they would be switched in how they appear in the file

Reply:
fixed. I think?